chapters/debugging.txt

```diff
@@ -68,9 +68,6 @@ ifdef::VK_NVX_device_generated_commands[]
 | ename:VK_OBJECT_TYPE_OBJECT_TABLE_NVX | slink:VkObjectTableNVX
 | ename:VK_OBJECT_TYPE_INDIRECT_COMMANDS_LAYOUT_NVX | slink:VkIndirectCommandsLayoutNVX
 endif::VK_NVX_device_generated_commands[]
-ifdef::VK_EXT_debug_utils[]
-| ename:VK_OBJECT_TYPE_DEBUG_UTILS_MESSENGER_EXT | slink:VkDebugUtilsMessengerEXT
-endif::VK_EXT_debug_utils[]
```

Comment:
Why is this removed?

Reply:
You told me to remove it since you were commiting this change in another submission.

Reply:
Right. Weird it shows in the diff then.
The change was merged, so I think you should rebase, just to be safe.

Reply:
Weird, looks like it got messed up even after a rebase.  I fixed it.